sound/soc/intel/skylake/skl-pcm.c

```diff
@@ -1476,7 +1476,7 @@ static const struct snd_soc_component_driver skl_component  = {
 	.ops		= &skl_platform_ops,
 	.pcm_new	= skl_pcm_new,
 	.pcm_free	= skl_pcm_free,
-	.ignore_module_refcount = 1, /* do not increase the refcount in core */
+	.module_get_upon_open = 1, /* increase the refcount when a pcm is opened */
```

Comment:
I wasn't crazy, this needs to be squashed with the previous patch to avoid bisect issues.